app/src/main/java/com/liato/bankdroid/WebViewActivity.java

```diff
@@ -78,6 +78,7 @@ public void onProgressChanged(WebView view, int progress) {
                     Runnable runnable = new Runnable() {
                         public void run() {
 //                            activity.hideProgressBar();
+                            if (mFirstPageLoaded) mWebView.setBackgroundColor(-1); // -1 == Color.WHITE
```

Comment:
http://developer.android.com/reference/android/graphics/Color.html#WHITE

I rather not `import static android.graphics.Color.WHITE;`and use `WHITE`(I think it is faster my way).